package.json

```diff
@@ -58,6 +60,8 @@
     "ember-source": "~3.13.0",
     "ember-source-channel-url": "^2.0.1",
     "ember-try": "^1.2.1",
+    "eslint": "^7.0.0",
+    "eslint-config-prettier": "^6.11.0",
```

Comment:
See other comments, but we either need to bring in the other cadre of prettier packages (or read through the details over in https://github.com/emberjs/rfcs/pull/628 😉) in this PR (`eslint-plugin-prettier` + `prettier`).